prow/autobump.sh

```diff
@@ -73,21 +75,23 @@ extract-commit() {
   echo ${c##*-}
 }
 
-old_version=$(git show HEAD:prow/cluster/plank_deployment.yaml | extract-version)
-version=$(cat cluster/plank_deployment.yaml | extract-version)
-comparison=$(extract-commit "${old_version}")...$(extract-commit "${version}")
-echo -e "Pushing $(color-version ${version}) to ${user}:autobump..." >&2
+oncall=$(python2.7 -c 'import urllib2;import json;print json.loads(urllib2.urlopen("https://storage.googleapis.com/kubernetes-jenkins/oncall.json").read())["Oncall"]["testinfra"]')
 
-title="Bump prow from ${old_version} to ${version}"
+old_prow_version=$(git show HEAD:prow/cluster/plank_deployment.yaml | extract-version)
+prow_version=$(cat cluster/plank_deployment.yaml | extract-version)
+prow_comparison=$(extract-commit "${old_prow_version}")...$(extract-commit "${prow_version}")
+echo -e "Pushing prow $(color-version ${prow_version}) to ${user}:autobump..." >&2
+
+title="Bump prow (${old_prow_version} to ${prow_version}) and test images as necessary"
 git add -A
-git commit -m "${title}"
+git commit -m "Bump all images"
```

Comment:
Can we please have a unique title? I guess you don't want to do the commit because this may differ. How about "Bump all images to latest as of $(date)" or something like that?

Ideally the commit history is not filled with hundreds of equivalent commit messages.